card/mod/03_machines/set/all/reset_machines.rb

```diff
@@ -0,0 +1,14 @@
+module ClassMethods
+  def reset_all_machines
+    Auth.as_bot do
+      Card.search(right: { codename: "machine_cache" }).each do |card|
+        card.update_columns trash: true
+        card.expire
+      end
+      Card.search(right: { codename: "machine_output" }).each do |card|
```

Comment:
not worth changing, but `{ codename: ["in", "machine_cache", "machine_output"] }` should also work.